build.gradle.kts

```diff
@@ -107,36 +118,98 @@ subprojects {
             testOptions {
                 unitTests.isReturnDefaultValues = true
             }
+
+            val sourcesJar by tasks.registering(Jar::class) {
+                from(sourceSets["main"].java.srcDirs)
+                classifier = "sources"
+            }
+
+            val doc by tasks.creating(Javadoc::class) {
+                isFailOnError = false
+                source = sourceSets["main"].java.sourceFiles
+                classpath += files(bootClasspath.joinToString(File.pathSeparator))
+                classpath += configurations.compile
+            }
         }
 
         configure<JacocoAndroidUnitTestReportExtension> {
             csv.enabled(false)
             html.enabled(true)
             xml.enabled(true)
         }
-
-        tasks.withType<Javadoc>().all { enabled = false }
     }
 
     if (!isSample) {
         apply {
-            plugin(BintrayRelease.plugin)
+            plugin(Release.MavenPublish.plugin)
+            plugin(Release.Bintray.plugin)
             plugin(Ktlint.plugin)
         }
 
-        configure<PublishExtension> {
-            artifactId = project.name
-            autoPublish = true
-            desc = "The easiest HTTP networking library in Kotlin/Android"
-            groupId = "com.github.kittinunf.fuel"
-            setLicences("MIT")
-            publishVersion = Fuel.publishVersion
-            uploadName = "Fuel-Android"
-            website = "https://github.com/kittinunf/Fuel"
-        }
-
         configure<KotlinterExtension> {
             reporters = arrayOf(ReporterType.plain.name, ReporterType.checkstyle.name)
         }
+
+        version = Fuel.publishVersion
+        group = Fuel.groupId
+        configure<BintrayExtension> {
```

Comment:
not sure if this could be replaced with 
```
bintray {
```

Reply:
Oh really? I will try!

Reply:
🙅‍♂️ I couldn't ... >_<

Reply:
https://github.com/bintray/gradle-bintray-plugin/issues/258 provides example that should work.